recipes/continuable/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class ContinuableConan(ConanFile):
+    name = "continuable"
+    description = "C++14 asynchronous allocation aware futures (supporting then, exception handling, coroutines and connections)"
+    topics = "asynchronous", "future", "coroutines", "header-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Naios/continuable"
+    license = "MIT"
+    settings = "os", "compiler"
+    no_copy_source = True
+    requires = (
+        "function2/4.1.0",
+    )
+    options = {
+        # Exceptions are disabled and `std::error_condition` is used as error_type. See tutorial-chaining-continuables-fail for details.
+        "with_no_exceptions": [True, False],
```

Comment:
I have 2 remarks:
- Do we want the `with_` prefix?
`with_` is currently used for including optional dependencies (e.g. `with_zlib`, `with_zstd`).
Isn't `no_exceptions`, `custom_error_type`, `unhandled_exceptions` better?
- I would invert `with_no_exceptions` to `with_exceptions`. This avoids a double negation.

Reply:
This is a good point.
The reason why i sticked to `with_` and `no`, is that it is exactly named like that as an option in the CMakeList of the project (`CTI_CONTINUABLE_WITH_NO_EXCEPTIONS`).

Iam open to change that of course.

Reply:
Applying my suggestions would make the it look like the other recipes.
But let's hear what *the others* have to say about this.

Reply:
@IceflowRE 
This (and the quotes remark) are my only comments.